.gitignore

```diff
@@ -1,3 +1,3 @@
-.idea
+.idea/
 vendor/
-dev/config.php
+etc/config.yml
```

Comment:
That´s where config files normally live under the [LSB](https://en.wikipedia.org/wiki/Linux_Standard_Base) :)